kratos/sources/model_part_io.cpp

```diff
@@ -4495,6 +4495,18 @@ TValueType& ModelPartIO::ExtractValue(std::string rWord, TValueType & rValue)
     return rValue;
 }
 
+bool& ModelPartIO::ExtractValue(std::string rWord, bool & rValue)
+{
+
+    if (rWord == "1" || rWord == "true" || rWord == "True") {
+        rValue = true;
+    } else {
+          rValue = false;
+    }
```

Comment:
```suggestion
    } else if (rWord == "0" || rWord == "false" || rWord == "False") {
        rValue = false;
    } else {
        KRATOS_ERROR << "Boolean argument could not be determined: " << rWord << std::endl;
    }
```

Probably even better what do you think?

Reply:
Certainly much better!